Makefile

```diff
@@ -1,4 +1,13 @@
-.PHONY: lint
+.PHONY: lint build install uninstall
 
 lint:
 	shellcheck scripts/*
+
+build:
+	docker build -t open-balena-installer -f installer/Dockerfile .
+
+install:
+	docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock open-balena-installer balena-install up -d --build
```

Comment:
We should be getting the host Docker socket from the environment by the time we merge this, right?